docs/UserGuide.md

```diff
@@ -384,56 +459,147 @@ Format: `c-clear`
+
+</div>
+
+Example:
+* Let's say after you familiarize yourself with tCheck, you want to remove all sample data in employee directory. But
+ you don't want to permanently delete those data, so that you can use them as a reference in the future. In this case
+ , you can follow these steps to archive all employees.
+
+Steps to archive all employees in the sample data:
+1. Type `c-archive-all` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message.
+2. All employees will be removed from Employee Directory inside the _Main View_.
+
+{Screenshot}
+
 #### 3.12 Listing all archived employees : `c-archive-list`
 
-Shows a list of all archived employees' contact details in tCheck.
+Shows a list of all archived employees in Employee Directory inside the _Main View_.
 
 Format: `c-archive-list`
 
-#### 3.13 Unarchiving an employee : `c-unarchive`
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about displaying archived employees in tCheck:**<br>
+
+Archived employees will be shown with a red color label, named as `[Archived]`, in front of their names. You may
+ refer to the screenshot below.<br>
 
-Unarchives the specified employee's contact detail from the tCheck's archived employee list.
+</div>
+
+Example:
+Let's say you want to see an archived employee's information, you can follow these steps.
+
+Steps to see archived employees:
+1. Type `c-archive-list` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message.
+2. All archived employees will be listed in Employee Directory inside the _Main View_.
+
+{Screenshot}
+
+#### 3.13 Unarchiving an employee : `c-unarchive`
+Unarchives the specified employee from the archived employee list.
 
 Format: `c-unarchive INDEX`
 
-* Unarchives the employee at the specified `INDEX`.
-* The index refers to the index number shown in the displayed `employees' contact details` panel.
-* The index **must be a positive integer** 1, 2, 3, ...
+* Unarchives the employee at the specified `INDEX` in the Employee Directory inside the _Main View_.<br>
+* The `INDEX` refers to the index number shown in the current Employee Directory inside the _Main View_.<br>
+* The `INDEX` **must be a positive integer** (e.g. 1, 2, 3, ...​).<br>
+* The `INDEX` **should not be larger than** the total number of employees listed in the current
+Employee Directory inside the _Main View_. For example, there are 2 employees listed in the current Employee
+ Directory inside the _Main View_. The `INDEX`should be less or equal to 2.<br>
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about Employee Directory:**<br>
+
+* If _Employee Directory_ currently shows a list of active/unarchived employees (Archived employees are displayed with a
```

Comment:
Similar for this, and other similar parts.

Reply:
Will remove it. Thanks!